CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 6.0.0
-
+- Added measurementId and appId to `FirebaseOptions` (both required for analytics)
```

Comment:
Please create and add this to `## 6.1.0`

Reply:
I've done this, now there's some conflicts in pubspec and changelog
when i forked the repository v7.0.0 was not there